site/docs/pages/getting-started.mdx

```diff
@@ -53,7 +53,7 @@ Obtain an API key from the [Coinbase Developer Platform APIs](https://www.coinba
 
 <img
   alt="OnchainKit copy API KEY"
-  src="https://onchainkit.xyz/assets/copy-api-key.png"
+  src="site/docs/public/assets/CDP_API_key_guide.png"
```

Comment:
I don't think this will work in production, you need `https://onchainkit.xyz/assets/` before the file name.

Also for future images, remember to keep the name of the file lower-case as it is the standard for URLs.